src/checkqueue.h

```diff
@@ -127,6 +127,9 @@ class CCheckQueue
     }
 
 public:
+    //! Mutex to ensure only one concurrent CCheckQueueControl
+    boost::mutex ControlMutex;
```

Comment:
Nit: Is there some coding standard for mutexes which makes you capitalize this ivar? If not, maybe `controlMutex` would be more consistent.